common/src/address.rs

```diff
@@ -46,6 +46,9 @@ const _ANYCAST_ADDRESS_INDEX: usize = 0;
 const DNS_ADDRESS_INDEX: usize = 1;
 const GZ_ADDRESS_INDEX: usize = 2;
 
+/// The maximum number of addresses per sled reserved for RSS.
+pub const RSS_RESERVED_ADDRESSES: u16 = 10;
```

Comment:
I shuffled this out of Nexus so it can be used / validated by RSS in a future PR.

Reply:
I'm missing some context here. What are these addresses for? Last I remember, RSS is on the bootstrap network, but I'm probably forgetting something.

Reply:
Also, I wonder if this should be named `NUM_RSS_RESERVED_ADDRESSES`, or similar for clarity.